views/js/admin/src/components/common/form/PsCheckbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="md-checkbox">
+    <label>
+      <slot>{{ $filters.translate(label) }}</slot>
+      <input
+        v-model="mutableValue"
+        type="checkbox"
+        class="form-control"
+        :disabled="disabled"
+        :value="value">
+      <i class="md-checkbox-control" />
+    </label>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { disabledProps } from '@/composables/props/disabledProps';
+import { useCheckboxModel } from '@/composables/props/model/useCheckboxModel';
+
+const { model, props, setup } = useCheckboxModel();
+
+export default defineComponent({
+  name: 'PsCheckbox',
+  model,
```

Comment:
Vanwaar de reden om dit direct op de scope te gooien? Wat geven deze composables terug en vanweer niet in de setup function?

Reply:
model, props en setup stuff, de eerste twee kunnen niet via setup erin